cliquet/__init__.py

```diff
@@ -327,3 +340,16 @@ def initialize_cliquet(config, version=None, project_name=None):
     # Include cliquet views with the correct api version prefix.
     config.include("cliquet", route_prefix=api_version)
     config.route_prefix = api_version
+
+
+def wrap_app(app, settings):
```

Comment:
I don't like too much the name of this method, should we name it ``plug_newrelic_if_activated``